exercises/bracket-push/lib/example.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:collection';
+
+class BracketPush {
+  bool isPaired(String input) {
+    ListQueue<String> stack = new ListQueue(input.length);
+
+    for (int i = 0; i < input.length; i++) {
+      if (input[i] == '(' || input[i] == '{' || input[i] == '[') {
+        stack.add(input[i]);
+      } else if (input[i] == ')' || input[i] == '}' || input[i] == ']') {
+        if (stack.isEmpty || !areMatching(stack.last, input[i])) {
+          return false;
+        } else {
+          stack.removeLast();
+        }
+      }
+    }
+
+    return stack.isEmpty;
+  }
+
+  bool areMatching(String opening, String closing) {
```

Comment:
Recommend setting areMatching as a private method. Though, it's not important, I think it helps highlight that `isPaired` is the primary method to use.